src/bin/index.ts

```diff
@@ -55,34 +55,31 @@ const parseCommandLineArguments = () => {
           defaultDescription: "The current working directory",
         })
       )
-      .command(Commands.New, "Creates a new CDK stack", (yargs) =>
-        yargs
-          .option("multi-app", {
-            type: "boolean",
-            description:
-              "Create the stack files within sub directories as the project defines multiple apps (defaults to false)",
-            default: false,
-          })
-          .option("init", {
-            type: "boolean",
-            description: "Create the cdk directory before building the app and stack files (defaults to true)",
-            default: true,
-          })
-          .option("app", {
-            type: "string",
-            description: "The name of your application e.g. Amigo",
-            demandOption: true,
-          })
-          .option("stack", {
-            type: "string",
-            description:
-              "The Guardian stack being used (as defined in your riff-raff.yaml). This will be applied as a tag to all of your resources.",
-            demandOption: true,
-          })
-          .option("yaml-template-location", {
-            type: "string",
-            description: "Path to the YAML CloudFormation template",
-          })
+      .command(
+        Commands.New,
+        "Creates a new CDK project within a `cdk` directory at the root of the repository",
```

Comment:
The description of this command is now a little longer. Long enough to trigger `prettier` to place it on a new line.